packages/app-desktop/gui/MasterPasswordDialog/Dialog.tsx

```
@@ -90,10 +90,12 @@ export default function(props: Props) {
 		// eslint-disable-next-line @seiyab/react-hooks/exhaustive-deps -- Old code before rule was applied
 	}, [currentPassword, password1, onClose, mode]);
 
+	// Show the "Re-enter password" confirmation field
 	const needToRepeatPassword = useMemo(() => {
```

Biome (2.4.6)

[error] This hook is being called from within a function or method that is not a hook or component.

(lint/correctness/useHookAtTopLevel)